src/domain/challenge/challenge.service.ts

```diff
@@ -68,7 +68,11 @@ export class ChallengeService {
     if (!challengeID)
       throw new Error(`Invalid challenge id passed in: ${challengeID}`);
     // Try to find the challenge
-    const challenge = await Challenge.findOne(challengeID);
+    const challenge = await this.challengeRepository.findOne({
+      where: { id: challengeID },
+      relations: ['groups'],
+    });
+    //const challenge = await Challenge.findOne(challengeID);
     if (!challenge) {
       const msg = `Unable to find challenge with ID: ${challengeID}`;
```

Comment:
Better to throw an error here, assuming you expect to have this information/message on the client.